### DISCUSSION TITLE
Auto-Crop Black Bars

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7321] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

How do I use the tool to auto-crop black bars from images? Does it have a filter to tell me how to accurately crop them out?

**Comment by fmw42** [comment]
Please post an image. Is the black on all 4 sides or just 1-3 sides?

`magick.exe input.suffix -bordercolor black -border 1 -fuzz XX% -trim +repage output.suffix`

See XX to some value near 0. Increase if you have some black left over.
